src/tbb/tools_api/disable_warnings.h

```diff
@@ -1,29 +1,23 @@
 /*
-    Copyright (c) 2005-2021 Intel Corporation
+  Copyright (C) 2005-2019 Intel Corporation
```

Comment:
@omalyshe Is it ok that copyrights have changed so much?

Reply:
@AnuyaWelling2801 The copyright year should be changed to xxx-**2022** (where xxx is the first year in the range in a particular file, leave it as-is).